docs/source/contrib/metrics.rst

```diff
@@ -24,23 +24,30 @@ metrics useful for regression tasks. Definitions of metrics are based on `Botchk
 .. _`Botchkarev 2018`:
         https://arxiv.org/ftp/arxiv/papers/1809/1809.03006.pdf
 
-Complete list of metrics:
-
-    - :class:`~ignite.contrib.metrics.regression.CanberraMetric`
-    - :class:`~ignite.contrib.metrics.regression.FractionalAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.FractionalBias`
-    - :class:`~ignite.contrib.metrics.regression.GeometricMeanAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.GeometricMeanRelativeAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.ManhattanDistance`
-    - :class:`~ignite.contrib.metrics.regression.MaximumAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.MeanAbsoluteRelativeError`
-    - :class:`~ignite.contrib.metrics.regression.MeanError`
-    - :class:`~ignite.contrib.metrics.regression.MeanNormalizedBias`
-    - :class:`~ignite.contrib.metrics.regression.MedianAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.MedianAbsolutePercentageError`
-    - :class:`~ignite.contrib.metrics.regression.MedianRelativeAbsoluteError`
-    - :class:`~ignite.contrib.metrics.regression.R2Score`
-    - :class:`~ignite.contrib.metrics.regression.WaveHedgesDistance`
+Complete list of metrics
```

Comment:
I know that this is a bit out of the scope of the PR, but I think it would be better to create a complete list for the whole module `ignite.contrib.metrics` vs only regression metrics. If you wish to do that in another PR, it is up to you.